sys/vfs/vfs.c

```diff
@@ -564,7 +564,7 @@ int vfs_umount(vfs_mount_t *mountp)
         return -EINVAL;
     }
     DEBUG("vfs_umount: -> \"%s\" open=%d\n", mountp->mount_point, atomic_load(&mountp->open_files));
-    if (atomic_load(&mountp->open_files) > 0) {
+    if (atomic_load(&mountp->open_files) > 0 || force) {
```

Comment:
Shouldn't this be
```suggestion
    if (atomic_load(&mountp->open_files) > 0 && !force) {
```

Maybe an `-EBUSY` test case would be good to properly test this ;-)